### DISCUSSION TITLE
Problem with signals that use the left mouse button

### DISCUSSION THREAD

**Discussion by WalbiGulbi** [peter-kish/gloot#178] [General]
Hi! Thank you for your work.

I don't know if it's a bug or I'm just stupid, but I have a problem with signals that use the left mouse button. I can use them once and inventory freezes – I can’t interact with it. This applies both to the existing inventory_item_activated signal (when, for example, I use it to transfer_autosplitmerge) and to my own signals. My own signals using RMB (e.g. shift+RMB) work perfectly.

I was able to reproduce this easily in your example (inventory_grid_stacked_ex_transfer.tscn) using the inventory_item_activated signal and split.

**Comment by peter-kish** [comment]
Yup, there seems to be a bug. I'll try to fix it for the next release.

Issue link: https://github.com/peter-kish/gloot/issues/179
